db/BaseService/index.js

```diff
@@ -0,0 +1,57 @@
+const assert = require("assert");
+const { knex } = require("../helpers");
+
+class BaseService {
+    hasRecord(object) {
+        assert.ok(typeof object === "object", "object<object> is required");
+
+        return knex(this.name)
+            .where(object)
+            .select("id")
+            .then((rows) => {
+                knex.destroy();
+
+                return Boolean(rows.length);
```

Comment:
although it results in the same. It is better to check whether length is !== 1 for clarity

Reply:
more efficient to use count for this thing, but do you really need this method at all?

Reply:
How is it more efficient to use count? We can talk about it tomorrow.

Reply:
more efficient since this will actually return all the rows deserialized and then you do a length on that array. Count OTH just sends just the integer from the dbserver so less network.